third_party/2and3/google/protobuf/message.pyi

```diff
@@ -15,6 +17,11 @@ class _ExtensionDict:
 
 _T = TypeVar("_T")
 
+if sys.version_info[0] == 2:
+    _Serialized = Union[bytes, buffer, unicode]
+else:
+    _Serialized = Union[bytes, memoryview]
```

Comment:
In case this also allows `bytestring`, this could be simplified to just `bytes`, per the [documentation](https://docs.python.org/3/library/typing.html#typing.ByteString).